sdk/containerregistry/Azure.Containers.ContainerRegistry/api/Azure.Containers.ContainerRegistry.netstandard2.0.cs

```diff
@@ -24,45 +24,41 @@ public partial class ContainerRepositoryClient
-        public bool CanRead { get { throw null; } set { } }
-        public bool CanWrite { get { throw null; } set { } }
-    }
-    [System.Runtime.InteropServices.StructLayoutAttribute(System.Runtime.InteropServices.LayoutKind.Sequential)]
-    public readonly partial struct PostContentSchemaGrantType : System.IEquatable<Azure.Containers.ContainerRegistry.PostContentSchemaGrantType>
-    {
-        private readonly object _dummy;
-        private readonly int _dummyPrimitive;
-        public PostContentSchemaGrantType(string value) { throw null; }
-        public static Azure.Containers.ContainerRegistry.PostContentSchemaGrantType AccessToken { get { throw null; } }
-        public static Azure.Containers.ContainerRegistry.PostContentSchemaGrantType AccessTokenRefreshToken { get { throw null; } }
-        public static Azure.Containers.ContainerRegistry.PostContentSchemaGrantType RefreshToken { get { throw null; } }
-        public bool Equals(Azure.Containers.ContainerRegistry.PostContentSchemaGrantType other) { throw null; }
-        [System.ComponentModel.EditorBrowsableAttribute(System.ComponentModel.EditorBrowsableState.Never)]
-        public override bool Equals(object obj) { throw null; }
-        [System.ComponentModel.EditorBrowsableAttribute(System.ComponentModel.EditorBrowsableState.Never)]
-        public override int GetHashCode() { throw null; }
-        public static bool operator ==(Azure.Containers.ContainerRegistry.PostContentSchemaGrantType left, Azure.Containers.ContainerRegistry.PostContentSchemaGrantType right) { throw null; }
-        public static implicit operator Azure.Containers.ContainerRegistry.PostContentSchemaGrantType (string value) { throw null; }
-        public static bool operator !=(Azure.Containers.ContainerRegistry.PostContentSchemaGrantType left, Azure.Containers.ContainerRegistry.PostContentSchemaGrantType right) { throw null; }
-        public override string ToString() { throw null; }
+        public bool? CanDelete { get { throw null; } set { } }
+        public bool? CanList { get { throw null; } set { } }
+        public bool? CanRead { get { throw null; } set { } }
+        public bool? CanWrite { get { throw null; } set { } }
     }
     public partial class RepositoryProperties
     {
         internal RepositoryProperties() { }
-        public System.DateTimeOffset CreatedOn { get { throw null; } }
-        public System.DateTimeOffset LastUpdatedOn { get { throw null; } }
+        public System.DateTimeOffset? CreatedOn { get { throw null; } }
+        public System.DateTimeOffset? LastUpdatedOn { get { throw null; } }
+        public string Name { get { throw null; } }
+        public string Registry { get { throw null; } }
+        public int? RegistryArtifactCount { get { throw null; } }
+        public int? TagCount { get { throw null; } }
+        public Azure.Containers.ContainerRegistry.ContentProperties WriteableProperties { get { throw null; } }
```

Comment:
Interesting - this is called WriteableProperties but the properties are actually read only. Does "Writeable" have special meaning in ACR?

Reply:
You can write to the members of `ContentProperties`.  `RepositoryProperties` itself is a readonly type.  Does that make sense to you @JoshLove-msft, or do you still have concerns?

Reply:
Ah ok. I misread the API. So what happens when you modify the properties? Is RepositoryProperties also an input type?

Reply:
RepositoryProperties is not an input type.  ContentProperties lets you set read/write/list/delete attributes on a resource (repository/manifest/tag).  I had originally called this ContentPermissions to be more specific, but there was a concern that given this is not using RBAC or ACLs, that the word permissions could be misleading.  It's possible they'll extend the ContentProperties collection with non-permission attributes as well.

Reply:
Got it, ContentProperties is input/output.